### PR TITLE
zh-translation:/docs/ops/setup/webhook/index.md

### PR DESCRIPTION
**Pull request by TomatoAres**
ref: https://github.com/servicemesher/istio-official-translation/issues/1054

### CODE REVIEW

File: content/zh/docs/ops/setup/webhook/index.md [hunk 1]
```diff
@@ -1,60 +1,41 @@
...
-webhook. With validating admission webhooks, you may reject requests
-to enforce custom admission policies. With mutating admission
-webhooks, you may change requests to enforce custom defaults.
+准入 Webhook 是 HTTP 方式的回调，接收准入请求并对其进行相关操作。可定义两种类型的准入 Webhook，Validating 准入 Webhook 和 Mutating 准入 Webhook。 使用 Validating Webhook，可以通过自定义的准入策略来拒绝请求；使用 Mutating Webhook，可以通过自定义默认值来修改请求。
 {{< /tip >}}
 
-Istio uses `ValidatingAdmissionWebhooks` for validating Istio
-configuration and `MutatingAdmissionWebhooks` for automatically
-injecting the sidecar proxy into user pods.
+Istio 使用 `ValidatingAdmissionWebhooks` 验证 Istio 配置，使用 `MutatingAdmissionWebhooks` 自动将 Sidecar 代理注入至用户 Pod
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/setup/webhook/index.md]:
```suggestion
Istio 使用 `ValidatingAdmissionWebhooks` 验证 Istio 配置，使用 `MutatingAdmissionWebhooks` 自动将 Sidecar 代理注入至用户 Pod。
```

File: content/zh/docs/ops/setup/webhook/index.md [hunk 2]
```diff
@@ -1,60 +1,41 @@
...
 
-See the [platform setup instructions](/docs/setup/platform-setup/)
-for Kubernetes provider specific setup instructions. Webhooks will not
-function properly if the cluster is misconfigured. You can follow
-these steps once the cluster has been configured and dynamic
-webhooks and dependent features are not functioning properly.
 
-1. Verify you’re using a supported version ({{< supported_kubernetes_versions >}}) of
-   [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and of the Kubernetes server:
+请参阅 [平台设置说明](/zh/docs/setup/platform-setup/)。如果群集配置错误，Webhook 将无法正常工作。集群配置后，当动态 Webhook 和相关特性不能正常工作时，你可以通过以下步骤进行检查。
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/setup/webhook/index.md]:
```suggestion
请参阅 [平台设置说明](/zh/docs/setup/platform-setup/)。如果集群配置错误，Webhook 将无法正常工作。集群配置后，当动态 Webhook 和相关特性不能正常工作时，你可以通过以下步骤进行检查。
```

File: content/zh/docs/ops/setup/webhook/index.md [hunk 3]
```diff
@@ -1,60 +1,41 @@
...
 
     {{< text bash >}}
     $ kubectl api-versions | grep admissionregistration.k8s.io/v1beta1
     admissionregistration.k8s.io/v1beta1
     {{< /text >}}
 
-1. Verify `MutatingAdmissionWebhook` and `ValidatingAdmissionWebhook` plugins are
-   listed in the `kube-apiserver --enable-admission-plugins`. Access
-   to this flag is [provider specific](/docs/setup/platform-setup/).
+3. 验证 `kube-apiserver --enable-admission-plugins` 配置中插件 `MutatingAdmissionWebhook` 和 `ValidatingAdmissionWebhook` 在是否被启用。通过检查[指定规范](/zh/docs/setup/install/kubernetes/#prerequisites)中的标志（`--enable-admission-plugins`）。
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/setup/webhook/index.md]:
```suggestion
3. 验证 `kube-apiserver --enable-admission-plugins` 配置中插件 `MutatingAdmissionWebhook` 和 `ValidatingAdmissionWebhook` 是否被启用。通过检查[指定规范](/zh/docs/setup/install/kubernetes/#prerequisites)中的标志（`--enable-admission-plugins`）。
```

File: content/zh/docs/ops/setup/webhook/index.md [hunk 4]
```diff
@@ -1,60 +1,41 @@
...
 
-1. Verify the Kubernetes api-server has network connectivity to the
-   webhook pod. e.g. incorrect `http_proxy` settings can interfere
-   api-server operation (see related issues
-   [here](https://github.com/kubernetes/kubernetes/pull/58698#discussion_r163879443)
-   and [here](https://github.com/kubernetes/kubeadm/issues/666) for more information).
+4. 验证 Kubernetes api-server 与 Webhook 所在 Pod 的网络连通是否正常。例如错误配置 `http_proxy` 可能干扰 api-server 正常运行（详细信息请参阅[pr](https://github.com/kubernetes/kubernetes/pull/58698#discussion_r163879443)和[issue](https://github.com/kubernetes/kubeadm/issues/666))。
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/setup/webhook/index.md]:
```suggestion
4. 验证 Kubernetes api-server 与 Webhook 所在 Pod 的网络连通是否正常。例如错误配置 `http_proxy` 可能干扰 api-server 正常运行（详细信息请参阅[pr](https://github.com/kubernetes/kubernetes/pull/58698#discussion_r163879443) 和 [issue](https://github.com/kubernetes/kubeadm/issues/666))。
```

File: content/zh/docs/ops/setup/webhook/index.md [hunk 5]
```diff
@@ -1,60 +1,40 @@
...
+1. 验证当前是否使用正确版本的 [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 和 kubernetes 服务
 
     {{< text bash >}}
     $ kubectl version --short
     Client Version: v1.10.2
     Server Version: v1.10.4-gke.0
     {{< /text >}}
 
-1. `admissionregistration.k8s.io/v1beta1` should be enabled
+2. `admissionregistration.k8s.io/v1beta1` 应是启用状态
```
Comment (tanjunchen) [inline content/zh/docs/ops/setup/webhook/index.md]: 2->1

File: content/zh/docs/ops/setup/webhook/index.md (hunk 3 above)
Comment (tanjunchen) [reply content/zh/docs/ops/setup/webhook/index.md]: 3->1

File: content/zh/docs/ops/setup/webhook/index.md [hunk 6]
```diff
@@ -1,60 +1,40 @@
...
-   listed in the `kube-apiserver --enable-admission-plugins`. Access
-   to this flag is [provider specific](/zh/docs/setup/platform-setup/).
+3. 验证 `kube-apiserver --enable-admission-plugins` 配置中插件 `MutatingAdmissionWebhook` 和 `ValidatingAdmissionWebhook` 在是否被启用。通过检查[指定规范](/zh/docs/setup/install/kubernetes/#prerequisites)中的标志（`--enable-admission-plugins`）。
 
-1. Verify the Kubernetes api-server has network connectivity to the
-   webhook pod. e.g. incorrect `http_proxy` settings can interfere
-   api-server operation (see related issues
-   [here](https://github.com/kubernetes/kubernetes/pull/58698#discussion_r163879443)
-   and [here](https://github.com/kubernetes/kubeadm/issues/666) for more information).
+4. 验证 Kubernetes api-server 与 Webhook 所在 Pod 的网络连通是否正常。例如错误配置 `http_proxy` 可能干扰 api-server 正常运行（详细信息请参阅[pr](https://github.com/kubernetes/kubernetes/pull/58698#discussion_r163879443)和[issue](https://github.com/kubernetes/kubeadm/issues/666))。
```
Comment (tanjunchen) [inline content/zh/docs/ops/setup/webhook/index.md]: 4->1

**Comment by tanjunchen** [comment]
@TomatoAres  squash commits

File: content/zh/docs/ops/setup/webhook/index.md (hunk 3 above)
Comment (tanjunchen) [reply content/zh/docs/ops/setup/webhook/index.md]: /zh/docs/setup/install/kubernetes/#prerequisites  this url is wrong. see en/ origin files

File: content/zh/docs/ops/setup/webhook/index.md (hunk 4 above)
Comment (tanjunchen) [reply content/zh/docs/ops/setup/webhook/index.md]: 4->1

**Comment by yuxiaobo96** [comment]
@rootsongjc 
帮合一下，时间太久了，本地的很多commit都没了，不太好squash

**Pull request merged**